docs/getting-started.md

```diff
@@ -0,0 +1,276 @@
+> NOTE: We have yet to publish our initial release to npm. However, once we do, the installation
```

Comment:
Great guide! 👍  The only thing missing is a reference to the fact that developers can use the sources in the packages instead of the transpiled resources, be it Sass or ES6. I don't think we need a whole step covering it, but it may be worth a mention in "Next Steps".